client/components/token-field/index.jsx

```diff
@@ -172,14 +172,22 @@ var TokenField = React.createClass( {
 
 	_renderInput: function() {
 		const { maxLength, value } = this.props;
+
+		let props = {
+			ref: 'input',
+			key: 'input',
+			disabled: this.props.disabled
+		};
+
+		if ( ! ( maxLength && value.length >= maxLength ) ) {
+			props = Object.assign( {}, props, {
+				value: this.state.incompleteTokenValue,
```

Comment:
Technically we could still pass this value in, yeah? I ask only because I was trying to grok the intent of this assignment, and eventually came to the conclusion that it's to prevent the `onChange` handler from being specified when `maxLength` is reached. If the code reflected this, I think it'd be easier to understand at a glance.